robottelo/hosts.py

```diff
@@ -1113,18 +1120,41 @@ def get_features(self):
         """Get capsule features"""
         return requests.get(f'https://{self.hostname}:9090/features', verify=False).text
 
+    def register_to_dogfood(self, ak_type='satellite'):
+        dogfood_canonical_hostname = settings.repos.dogfood_repo_host.partition('//')[2]
+        # get hostname of dogfood machine
+        dig_result = self.execute(f'dig +short {dogfood_canonical_hostname}')
+        # the host name finishes with a dot, so last character is removed
+        dogfood_hostname = dig_result.stdout.split()[0][:-1]
+        dogfood = Satellite(dogfood_hostname)
+        self.install_katello_ca(satellite=dogfood)
+        # satellite version consist from x.y.z, we need only x.y
+        sat_release = '.'.join(self.version.split('.')[:2])
+        rhel_release = settings.server.version.rhel_release
+        cmd_result = self.register_contenthost(
+            org=f'{settings.subscription.dogfood_org}',
+            activation_key=f'{ak_type}-{sat_release}-qa-rhel{rhel_release}',
```

Comment:
nitpick:
we already keep this in `subscription.yaml`, however it might use update, as it's still hardcoeded to `rhel7`:

```
CAPSULE_DOGFOOD_ACTIVATIONKEY: '@format capsule-{this.robottelo.satellite_version}-qa-rhel7'
```

Reply:
oh, and unused anywhere. Update might look like
```
CAPSULE_DOGFOOD_ACTIVATIONKEY: '@format capsule-{this.robottelo.satellite_version}-qa-rhel{this.server.version.rhel_release}'
```